src/test/ui/resolve/tuple-struct-alias.stderr

```diff
@@ -2,31 +2,29 @@ error[E0423]: expected function, found self type `Self`
   --> $DIR/tuple-struct-alias.rs:16:17
    |
 16 |         let s = Self(0, 1); //~ ERROR expected function
-   |                 ^^^^ did you mean `Self { /* fields */ }`?
+   |                 ^^^^ not a function
+   |
+   = note: can't instantiate `Self`, you must use the implemented struct directly
```

Comment:
This message would be better if it mentioned the struct `S`. It'd be even better if it checked that it was a suggestion:

```
  = help: can't instantiate `Self`, you must use the struct `S` directly:
   |
16 |         let s = S(0, 1); //~ ERROR expected function
   |                 ^